history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
+-  rendering setting fixes for copy and paste actions and floating point
+   rendering
+-  image interpolation can now be turned on and off in the clients
+-  more readable timestamps
+-  measurement table fixes
+-  directories are now preserved when downloading multiple original files
+-  various improvements to the imageJ plugin's handling of ROIs and
+   measurement tool functionality when interacting with OMERO
+-  current session key can now be returned via the CLI
+-  other CLI improvements including reporting for 'delete' and 'chgrp',
+   usability of 'chmod' for downgrading group permissions, and listing empty
+   tagsets
+-  better support for groups in OMERO.matlab methods
```

Comment:
No strong opinion on this as I can see it either way. I would be slightly more definitive and say we added or exposed group support in these methods.